datadog_checks_base/datadog_checks/base/utils/db/sql.py

```diff
@@ -0,0 +1,13 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import mmh3
+
+
+def compute_sql_signature(normalized_query):
+    """
+    Given an already obfuscated & normalized SQL query, generate its 64-bit hex signature.
+    """
+    if not normalized_query:
+        return None
+    return format(mmh3.hash64(normalized_query, signed=False)[0], 'x')
```

Comment:
Since this hash calculation must match the backend approach, adding a comment here would be helpful for future maintainers in case they get the wild idea of changing the algorithm to something "new".